### PR TITLE
upgrading to latest inputs

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3497,7 +3497,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#ad0396971c07c13556a9ea79d399be39c9a2d4ed",
+      "version": "github:BrightspaceHypermediaComponents/activities#240f96738301df47bfe3c825ec6d4a864f1dcfa6",
```
Comment (dlockhart) [inline package-lock.json]: FYI @ryantmer

File: package-lock.json [hunk 2]
```diff
@@ -4202,7 +4203,7 @@
       }
     },
     "d2l-rubric": {
-      "version": "github:Brightspace/d2l-rubric#ad22c14bb295871aac1ba4a0aa7d5e5085f4c360",
+      "version": "github:Brightspace/d2l-rubric#74b3ab85e1b9c5b331abaef9a1f6290918059f78",
```
Comment (dlockhart) [inline package-lock.json]: FYI @danyakarras

File: package-lock.json [hunk 3]
```diff
@@ -4272,27 +4273,8 @@
...
-        "d2l-offscreen": "github:BrightspaceUI/offscreen#semver:^4",
-        "d2l-polymer-siren-behaviors": "github:Brightspace/polymer-siren-behaviors#semver:^1",
-        "d2l-progress": "github:BrightspaceUI/progress#semver:^1",
-        "d2l-typography": "github:BrightspaceUI/typography#semver:^7",
-        "siren-entity": "github:BrightspaceHypermediaComponents/siren-entity#semver:^1"
-      }
-    },
     "d2l-sequence-viewer": {
-      "version": "github:Brightspace/d2l-sequence-viewer#90ac5e69bcf592007a7f393914f834caa065e3ee",
+      "version": "github:Brightspace/d2l-sequence-viewer#60ac08927198002715e261f57f8c46ec21b9100b",
```
Comment (dlockhart) [inline package-lock.json]: FYI @ChrisKraljevicD2L... and the loss of `d2l-sequence-navigator` is I'm assuming 💯 ?

File: package-lock.json (hunk 1 above)
Comment (ryantmer) [reply package-lock.json]: Was about to do this anyway 😄 Thanks!

**Pull request merged**